core/controllers/admin.py

```diff
@@ -2139,8 +2141,8 @@ def delete(self) -> None:
         """Initiates the pre-deletion process for a user.
 
         Raises:
-            InvalidInputException. The username doesn't belong to any user.
-            InvalidInputException. The user ID retrieved from the username
+            PageNotFoundException. The username doesn't belong to any user.
+            UnauthorizedUserException. The user ID retrieved from the username
```

Comment:
Probably this is still 400, we don't expect it to happen? It means the various fields of the request are inconsistent.